### PR TITLE
Feature/project components

### PR DESCRIPTION
**Pull request by vechicin**
# In this feature I did the following: 

- Create initial project UI
- Use React class component

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      <button type="button">3</button>
+      <button type="button" className="operator">+</button>
+      <button type="button" className="occupy-two">0</button>
+      <button type="button">.</button>
+      <button type="button" className="operator">=</button>
+    </div>
+  );
+}
+
+export default Calculator;
```
Comment (DeliceLydia) [inline src/components/Calculator.js]: - [ ] The `Calculator.js` component should be class-based please work on that

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @vechicin, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! my major highlights for your project:
- PR is descriptive
- UI design looks exactly like the design 
- No linters errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- in the `public` folder you can keep `index.html` only that what you need for now


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (vechicin) [reply src/components/Calculator.js:30]: Screen Shot 2022-01-04 at 15 07 46

# Hello @DeliceLydia 
I did exactly that at first but I was getting this linters error, I don't know what to do at this stage. Could you please point why am I getting that error?

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉

## Hi @vechicin, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**